### PR TITLE
Update pk.srt

### PR DESCRIPTION
**Pull request by toinhoalam**

### CODE REVIEW

File: musi/pk.srt [hunk 1]
```diff
@@ -4556,35 +4556,48 @@ point. let it open then.
...
-then also i decided atleast once in this life,
+ni la tenpo wan pi lon mi anu tenpo mute la 
+mi weka e tenpo mi. olin li weka tenpo.
+==then also i decided atleast once in this life,
 i will waste my time. love is a waste of time.
 
 983
 01:50:32,436 --> 01:50:43,378
-many time i see mirror and do my hair.
+tenpo mute la mi lukin e mi li pona e linja mi.
```
Comment (stefichjo) [inline musi/pk.srt]: According to pu, one cannot simply omit the first "li" of a sentence with "mi" as a subject. Instead, one has to form multiple sentences with "mi" (each without "li").

Please:
→ mi pona e linja mi.

File: musi/pk.srt [hunk 2]
```diff
@@ -4556,35 +4556,48 @@ point. let it open then.
...
+==many time i see mirror and do my hair.
 
 984
 01:50:43,759 --> 01:50:55,759
-may time i changed my cloths and put deodrant.
+tenpo mute la mi ante e len mi li telo pi kon pona e mi.
```
Comment (stefichjo) [inline musi/pk.srt]: → mi telo pi kon pona e mi.

**Review by stefichjo — CHANGES_REQUESTED** [review]

File: musi/pk.srt (hunk 1 above)
Comment (toinhoalam) [reply musi/pk.srt]: (One cannot put a second li in a sentence with mi as subject?) pona. mi sona ala e ni. It also seems to fit better the song. In my subtitles (in Portuguese) there is one sentence for each action he does many times.

**Review by stefichjo — APPROVED** [review]
Maybe repeating "tenpo mute la" wasn't necessary. But we can talk about this later. :)

**Pull request merged**